src/components/Dashboard.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div> 
+    <table  align="center">
+      <tr v-for="(user, key) in doneCurrent" :key="key">
+        <td class="name">{{user.name}}さんようこそ！！</td>
```

Comment:
マスタッシュの両内側はスペースをあけるのが通例です。
エディターの自動整形を利用しましょう。
```suggestion
        <td class="name">{{ user.name }}さんようこそ！！</td>
```

Reply:
スペースいれました。